sonar-python-plugin/src/main/resources/org/sonar/plugins/python/flake8/rules.json

```diff
@@ -0,0 +1,600 @@
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+  {
+    "key": "E731",
+    "name": "do not assign a lambda expression, use a def",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+  {
+    "key": "E741",
+    "name": "do not use variables named ‘l’, ‘O’, or ‘I’",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+  {
+    "key": "E742",
+    "name": "do not define classes named ‘l’, ‘O’, or ‘I’",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+  {
+    "key": "E743",
+    "name": "do not define functions named ‘l’, ‘O’, or ‘I’",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+
+  {
+    "key": "E901",
+    "name": "SyntaxError or IndentationError",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+  {
+    "key": "E902",
+    "name": "IOError",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+
+  {
+    "key": "W191",
+    "name": "indentation contains tabs",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+
+  {
+    "key": "W291",
+    "name": "trailing whitespace",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+  {
+    "key": "W292",
+    "name": "no newline at end of file",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+  {
+    "key": "W293",
+    "name": "blank line contains whitespace",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+
+  {
+    "key": "W391",
+    "name": "blank line at end of file",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+
+  {
+    "key": "W503",
+    "name": "line break before binary operator",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+  {
+    "key": "W504",
+    "name": "line break after binary operator",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+  {
+    "key": "W505",
+    "name": "doc line too long (82 > 79 characters)",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+
+  {
+    "key": "W601",
+    "name": ".has_key() is deprecated, use ‘in’",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+  {
+    "key": "W602",
+    "name": "deprecated form of raising exception",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+  {
+    "key": "W603",
+    "name": "‘<>’ is deprecated, use ‘!=’",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+  {
+    "key": "W604",
+    "name": "backticks are deprecated, use ‘repr()’",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+  {
+    "key": "W605",
+    "name": "invalid escape sequence ‘x’",
+    "url": "https://flake8.pycqa.org/en/latest/user/error-codes.html"
+  },
+  {
+    "key": "W606",
```

Comment:
They come from https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes for which there is a link at the bottom of the Flake8 page.
We decided to keep the generic message for importing external issues, which links to `{linter_name}` website, so to be consistent I kept Flake8 website there.
I could use a specific message and a different link for those rules though, do you think it would be better that way?